src/build/build.ts

```diff
@@ -26,6 +24,10 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
+// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
+// entity. Upgrade to import once compatible .d.ts file is released.
+import mergeStream = require('merge-stream');
```

Comment:
I get this error when using `@types/merge-stream`:
```
src/build/build.ts(29,30): 
  error TS2497: 
  Module '"/Users/fschott/Code/polytool/node_modules/@types/merge-stream/index"' 
  resolves to a non-module entity and cannot be imported using this construct.
```

I don't think it's worth writing a custom declaration now, but I'll re-write the TODO to mention that one could be added if someone does refactor some of this code.